File: java-solutions/Popsicles.java
import java.util.Scanner;

public class Popsicles
{
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int siblings = input.nextInt();
        int popsicles = input.nextInt();

        if (popsicles % siblings == 0) {
            System.out.println("give away");
        } else {
            System.out.println("eat them yourself");
        }
        input.close();
    }
}

File: cpp-solutions/popsicles.cpp
#include <iostream>

using namespace std;

int main() {
    int siblings, popsicles;

    cin >> siblings;
    cin >> popsicles;

    if (popsicles % siblings == 0) {
        cout << "give away";
    } else {
        cout << "eat them yourself";
    }

    return 0;
}